Library/Homebrew/system_config.rb

```diff
@@ -74,6 +74,13 @@ def describe_homebrew_ruby_version
       end
     end
 
+    def system_ruby_version
+      out, _, status = system_command("/usr/bin/ruby", args: ["-e", "p RUBY_VERSION"], print_stderr: false)
```

Comment:
```suggestion
      out, _, status = system_command("/usr/bin/ruby", args: ["-e", "puts RUBY_VERSION"], print_stderr: false)
```

This should remove the need for the `.chomp`